content/en/docs/release-oversight/risk-analysis.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Risk Analysis"
+description: Overview of a Test Failure Risk Analysis.
+---
+
+### Summary
+In an effort to raise awareness for potential regressions in tests with a high pass rate, 
+tooling to analyze the overall risk for individual test failures has been created.  Risk 
+Analysis compares failed tests in periodics against the historical results of similar periodics and 
```

Comment:
Should the first periodics be presubmits?

Reply:
Good question and I'm open to reworking it.  I went with periodics since we do analysis on [nightly jobs](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-sdn-upgrade/1681581667443019776) as well as presubmits.  So I was thinking periodics covered both.

Reply:
Presubmits aren't periodics

Reply:
Open to suggestions, just remove periodics and say:

Analysis compares failed tests against the historical results of similar (is periodics ok here or other?)

Or if I use presubmits are we ok that the nightly / ci analysis isn't really against presubmits?

Reply:
`Analysis compares failed tests against the historical results of similar periodics` makes sense, you only do the comparison against periodics IIRC?

Reply:
Analysis isn't limited to only presubmits though.

Reply:
> Analysis isn't limited to only presubmits though.

Hm... Is the doc about risk analysis specifically for PRs or more?

Reply:
ah, so risk analysis is also done on periodics against historical data of the periodic.